cron/correlateUserMacAddress.py

```diff
@@ -134,7 +134,7 @@ def main():
 
     es = ElasticsearchClient((list('{0}'.format(s) for s in options.esservers)))
     # create intelligence index if it's not already there
-    es.create_index('intelligence', ignore_fail=True)
+    es.create_index('intelligence')
```

Comment:
This change was merged into master, so I cherry-picked it here since we're technically dependent on it.